tensorflow-core/pom.xml

```diff
@@ -141,6 +143,7 @@
         <javacpp.platform.linux-x86>${javacpp.platform}</javacpp.platform.linux-x86>
         <javacpp.platform.linux-x86_64>${javacpp.platform}</javacpp.platform.linux-x86_64>
         <javacpp.platform.macosx-x86_64>${javacpp.platform}</javacpp.platform.macosx-x86_64>
+        <javacpp.platform.macosx-arm64>${javacpp.platform}</javacpp.platform.macosx-arm64>
```

Comment:
It'd be more consistent with the linux-* ones to put anything macosx-arm64 before macosx-x86_64.